Rubberduck.RibbonDispatcher/Abstract/Interfaces.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+
+using Rubberduck.RibbonDispatcher;
+
+// I have left these three collected together for now, until I complete the design and implementation
+// of their concrete classes - as a convenience and reminder.
+namespace Rubberduck.RibbonDispatcher.Abstract
+{
+    public interface IRibbonText : IReadOnlyList<IRibbonTextLanguage>
+    {
+        string DefaultLangCode { get; }
+    }
+
+    /// <summary>All the control-specific {IRibbonTextLanguageControl} for a language.</summary>
+    public interface IRibbonTextLanguage : IReadOnlyList<IRibbonTextLanguageControl>
+    {
+        string LangCode         { get; }
```

Comment:
why do you preserve aligned indentation across different types?

Reply:
I find the ability to vertically scan aligned text a real boon in reading code later. It also visually emphasizes that all the items in an alignment are following a single pattern, with only the names changing. Coming back months later to read the code, the visual emphasis of "sameness" is  immediately  recognizable.